README.md

```diff
@@ -192,68 +190,30 @@ or
 
 ```
 MyEventModel = Model.extend({
-    behaviorGroup: Ember.computed(
-        'event',
-        function() {
-            return this.get( 'event' );
-        }
-    )
+    canCancel: Ember.computed( function () {
+        return false;
+    })
 });
```

Comment:
Do we want to add a third example of just a function that can be executed and returns a boolean value?  The computed example we have is essentially that already and wondering if want to show computed example with a property it is observing.  How we have it should make sense for those familiar with Ember but for those that are not a more inclusive example may be a better presentation.